ground/gcs/src/plugins/coreplugin/alarmsmonitorwidget.cpp

```diff
@@ -41,7 +41,7 @@ void AlarmsMonitorWidget::init(QSvgRenderer *renderer,QGraphicsSvgItem *graph)
     error_sym->setParentItem(graph);
     error_txt = new QGraphicsTextItem();
     error_txt->setDefaultTextColor(Qt::white);
-    error_txt->setFont(QFont("Helvetica",22,2));
+    error_txt->setFont(QFont("Helvetica",24,2));
```

Comment:
Bunch of missing spaces between args and operators in this commit (please write new code to current standards unless it's ridiculously ugly to).